example.main.scss

```diff
@@ -2,7 +2,7 @@
    INUITCSS
    ========================================================================== */
 
-/*!*
```

Comment:
This is a different question/discussion. I think it’s worth fixing the comment style as we agreed on an official commenting-style anyways and this one doesn’t match it.

Can we move the inuit credit comment discussion into a separate ticket perhaps?